plugins/experimental/icap/icap_plugin.cc

```diff
@@ -28,6 +28,7 @@
 #include <string>
 #include <regex>
 
+#include <inttypes.h>
```

Comment:
Kind of nitpicking, but `<inttypes.h>` is deprecated. `<cinttypes>` is a replacement. 
 
> `<inttypes.h>`(C++11)(deprecated) | Behaves as if each name from `<cinttypes>` is placed in global namespace
https://en.cppreference.com/w/cpp/header%23C_compatibility_headers

Reply:
Thanks! I've amended the change.